### ISSUE TITLE
Don't use google fonts

### ISSUE THREAD

**Issue by sylae** [openplanet-nl/tmio#60]
muh privacy

![image](https://github.com/openplanet-nl/tmio/assets/2791628/8b0e8ec9-1525-4bf6-94b7-5767a16d5c69)

surely a locally-hosted version wouldn't be a huge overhead, esp as it's only for the "Trackmania.io" heading text?

**Comment by codecat** [comment]
Yeah, we can do that. Dunno how big the font is but it should be fine to host this one ourselves!
